pkg/didcomm/packer/jwe/authcrypt/authcrypt_test.go

```diff
@@ -331,10 +331,14 @@ func TestEncrypt(t *testing.T) {
 			[][]byte{base58.Decode(recSign)})
 		require.NoError(t, err)
 
-		msgOut, sendKey, err := recPacker.Unpack(enc)
+		msgOut, sendKey, recKey, err := recPacker.Unpack(enc)
 		require.NoError(t, err)
 		require.Equal(t, msgIn, msgOut)
 		require.Equal(t, base58.Encode(sendKey), encSign)
+		// this won't work, since input keys and output keys to this Packer
```

Comment:
what’s the resolution to this comment?

Reply:
The JWE packer would need to be changed to use the same key format as `Pack()` parameters and `Unpack()` return values, so the keys returned by `Unpack()` are keys passed into `Pack()` rather than the Curve25519 conversions of given Ed25519 keys. With an agent that supports multiple formats for encryption keys, we could do this - enforce that the JWE packer uses Curve25519 keys as parameters/return values, and that it only be chosen for DIDComm channels that have Curve25519 pubkeys available on both ends.

Reply:
Is this a TODO?